### ISSUE TITLE
Improve mobile experience

### ISSUE THREAD

**Issue by sprx97** [Spartan97/OldTimeHockey#102]

**Comment by chrisparsons83** [comment]
I don't mind helping out with this, I could probably do this while working on #98.

What would be the list of things to tackle?

- Fix sizing of adsense
- Collapsible Menu
- Leaderboard table scales to fit, potentially a better horizontal scroll?
- Standings table scale to fit
- Hall of fame drops down to one per row, instead of two per row?

**Comment by sprx97** [comment]
The collapsing menu and better table scaling/collapsing (both in Standings and Leaderboard) are the big ones for me.

The HOF is nice to have and I'm not reproing the adsense issue but if it's annoying go for it.

**Comment by sprx97** [comment]
FWIW half our sessions this season have come from mobile devices so that's what's driving my hope to improve this. What people do most is check the Leaderboard page (70+% of pageviews) followed by homepage (presumably to navigate to elsewhere) and Standings at just over 10% each. Those three pages should be our prio, along with the navbar/menu.

**Comment by chrisparsons83** [comment]
Yeah that menu is kind of silly for the amount of real estate it takes up. I imagine some sort of flyout would be preferred. What do you think about https://react.semantic-ui.com/modules/sidebar/ as an option?

**Comment by sprx97** [comment]
Works for me. Probably no images on the buttons though, just the labels. That looks pretty tacky unless we have a professional designer working on it IMO.

**Comment by scottij1** [comment]
This can probably be closed @sprx97 I updated the legacy (current) site nav to be mobile friendly a few months ago, and the new site will be mobile first.
